pennylane/templates/subroutines/qubitization.py

```diff
@@ -150,7 +150,11 @@ def compute_decomposition(*_, **kwargs):  # pylint: disable=arguments-differ
 
         decomp_ops = []
 
-        decomp_ops.append(qml.Reflection(qml.Identity(control)))
+        # QJIT is not compatible with qml.Identity(control), so we build the
+        # identity as a tensor product instead
```

Comment:
No need to put this comment here. Instead, we should create an issue for Catalyst to support `qml.Identity(list_of_wires)`.

Reply:
I already submitted an issue [here](https://github.com/PennyLaneAI/catalyst/issues/1175).